geps/gep-1742.md

```diff
@@ -1,7 +1,7 @@
 # GEP-1742: Timeouts
 
 * Issue: [#1742](https://github.com/kubernetes-sigs/gateway-api/issues/1742)
-* Status: Provisional
+* Status: Implementable
```

Comment:
The more I think about it the more I feel like we should merge this PR in a `Provisional` state to allow just a bit more soak time, and to invite others to bring their own PRs to the table prior to when we would start implementation. We could bring the GEP up at the next couple of weekly syncs just to let people know now's a good time to chime in, and maybe add a few commits and ideas of their own.

```suggestion
* Status: Provisional
```

Let me know your thoughts?

Reply:
That's a good idea. Done.

Reply:
Thank you, I'm adding this GEP to the next couple weeks of agenda to provide reminders that we're just about ready to move to `Implementable` and now is a critical time for anyone interested who hasn't yet jump in to jump in. :+1: